### PR TITLE
CRAN Package `BART`

### PR DESCRIPTION
**Pull request by Gattocrucco**
I followed the instructions on [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper) to create a recipe for the CRAN package `BART`.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-bart/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BART_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BART/BART_{{ version }}.tar.gz
+  sha256: 6e23eac476e894c0e6540eaa80e8436ea04261ce6aaac8d4f6f8f7ceeb082e0f
+
+build:
+  merge_build_host: True  # [win]
```
Comment (mfansler) [inline recipes/r-bart/meta.yaml:16]: Windows workaround (see https://github.com/conda/conda-build/pull/4786):

```suggestion
  merge_build_host: True  # [win]
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
```

File: recipes/r-bart/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```
Comment (mfansler) [inline recipes/r-bart/meta.yaml:38]: OpenMP declarations go in `build` (they have *strong_exports*):

```suggestion
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
    - libgomp                      # [linux]
    - llvm-openmp                  # [osx]
```

File: recipes/r-bart/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    - r-nnet
+    - r-survival
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp >=0.12.3
+    - r-nlme
+    - r-nnet
+    - r-survival
+    - llvm-openmp
```
Comment (mfansler) [inline recipes/r-bart/meta.yaml:52]: The *strong* exports will inject this automatically.

```suggestion
```

File: recipes/r-bart/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+test:
+  commands:
+    - $R -e "library('BART')"           # [not win]
+    - "\"%R%\" -e \"library('BART')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=BART
+  license: GPL-2.0-only
```
Comment (mfansler) [inline recipes/r-bart/meta.yaml:61]: DESCRIPTION shows `GPL (>= 2)`

```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-bart/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  summary: Bayesian Additive Regression Trees (BART) provide flexible nonparametric modeling
+    of covariates for continuous, binary, categorical and time-to-event outcomes.  For
+    more information see Sparapani, Spanbauer and McCulloch <doi:10.18637/jss.v097.i01>.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-bart/meta.yaml:67]: Might as well package both (similar to CRAN):

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this! ❤️ 

Here are some changes to help get it building.

**Comment by Gattocrucco** [comment]
Thanks!

The bot says to ping @conda-forge/r, how do I do that?

**Comment by mfansler** [comment]
> Thanks!
> 
> The bot says to ping @ conda-forge/r, how do I do that?

I'm on the team 😉  We can safely ignore that linter failure - [it's a bug](https://github.com/conda-forge/conda-smithy/issues/1817).

**Review by mfansler — APPROVED** [review]
Looks good to me!

@Gattocrucco was there anything else you were planning to edit? Otherwise, seems ready to merge.

**Comment by Gattocrucco** [comment]
Nope, all ready.

**Pull request merged**